accord-core/src/main/java/accord/api/Write.java

```diff
@@ -30,5 +31,5 @@
  */
 public interface Write
 {
-    AsyncChain<Void> apply(Seekable key, SafeCommandStore safeStore, Timestamp executeAt, DataStore store);
+    AsyncChain<Void> apply(Seekable key, SafeCommandStore safeStore, Timestamp executeAt, DataStore store, Command.Executed command);
```

Comment:
Do we need to provide the whole command, or could we provide just the `PartialTxn` or `Update`?